Fitness/Views/Headers/FacilityHoursHeaderView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  FacilityHoursHeaderView.swift
+//  Fitness
+//
+//  Created by Ji Hwan Seung on 4/24/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class FacilityHoursHeaderView: UITableViewHeaderFooterView {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.facilityHoursHeaderView
+    var iconImageView: UIImageView!
+    var downArrow: UIImageView!
+    var facilityNameLabel: UILabel!
+    var upArrow: UIImageView!
+    var statusLabel: UILabel!
+    var todayTimeLabel: UILabel!
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        
+        facilityNameLabel = UILabel()
+        facilityNameLabel.font = ._16MontserratMedium
+        facilityNameLabel.textColor = .fitnessBlack
+        facilityNameLabel.sizeToFit()
+        facilityNameLabel.textAlignment = .left
+        facilityNameLabel.text = ""
```

Comment:
i think its unnecessary to set the text to `""` unless you need it to be that for a specific reason